README.md

```diff
@@ -42,16 +42,26 @@ JCodec is free software distributed under FreeBSD License.
 Build from the source and include both JARs (jcodec.jar and jcodec-javase.jar) in your projects. Alternatively, obsolete
 versions can be included automatically with maven. For this just add below snippet to your pom.xml-
 
+For JDK
 ```xml
 <dependency>
```

Comment:
we should add transitive deps in `jcodec-javase` and `jcodec-android` in the `0.2.1` release